Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "mathbench"
-version = "0.3.1"
+version = "0.3.2"
 authors = ["Cameron Hart <cameron.hart@gmail.com>"]
 edition = "2018"
 
 [features]
 # defaults can be disabled for benchmarks but are required for tests
-default = ["cgmath", "euclid", "nalgebra", "vek"]
+default = ["cgmath", "euclid", "nalgebra", "vek", "static-math"]
```

Comment:
I generally only add well known libraries to the defaults. It's easy enough to compile it in by building with `--features static-math`. `pathfinder_geometry` is the main other library here that isn't a default.

Reply:
Yeah sure !!! I only want the output of the python script only for curiosity because i have a very old machine maybe you could upload as a text file
Maybe you could do a ranking of the best 6 libs, but is fine don't worry. I think my implementation serves as a parameter since it is the only one that does not use unsafe and SIMD(explicitly)

Reply:
I'll include all libraries next time I run the benchmarks. I'm mostly trying to keep defaults limited since it means a long compile for someone who just downloads mathbench and wants to check it out.